### PR TITLE
added singularity file

### PR DESCRIPTION
**Pull request by lucadelu**
Support for singularity container technology (https://wiki.debian.org/singularity)

### CODE REVIEW

File: singularity/debian/singularityfile_debian [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+        zlib1g-dev
+
+    # Set Python 3 as default Python version
+    update-alternatives --install /usr/bin/python python /usr/bin/python3 1
+    echo LANG="en_US.UTF-8" > /etc/default/locale
+    # compile GRASS
+    cd /tmp/grass_build
+    # -O2 -march=native -std=gnu99 -m64
+    LDFLAGS="-s"
+    CFLAGS="-O2 -march=x86-64 -std=gnu99 -m64"
```
Comment (neteler) [inline singularity/debian/singularityfile_debian]: Not sure if specifying `-march=x86-64` is a good idea

File: singularity/debian/singularityfile_debian [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+        python3-dateutil \
+        python3-dev \
+        python3-gdal \
+        python3-magic \
+        python3-numpy \
+        python3-pdal \
+        python3-pil \
+        python3-pip \
+        python3-ply \
+        python3-requests \
```
Comment (neteler) [inline singularity/debian/singularityfile_debian]: AFAIK python3-requests is not used in GRASS GIS

File: singularity/debian/singularityfile_debian (hunk 1 above)
Comment (lucadelu) [reply singularity/debian/singularityfile_debian]: Removed

File: singularity/debian/singularityfile_debian (hunk 2 above)
Comment (lucadelu) [reply singularity/debian/singularityfile_debian]: But it is required from some addons (i.modis and also i.sentinel) that the script install.

**Pull request merged**